app/controllers/api/v1/dqt_records_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class DqtRecordsController < Api::ApiController
+      include ApiTokenAuthenticatable
+
+      def show
+        hash = Dqt::Client.new.api.dqt_record.show(params: { teacher_reference_number: params[:id].to_i })
```

Comment:
Are we sure TRNs can't have leading zeros? What do we gain by `to_i`

Reply:
from the sample of data I have I don't see any with leading zero but that may be a false assumption. i just thought it would minimise the chance of someone putting in dodgee data. I will remove it